cmd/training-operator.v1/main.go

```diff
@@ -165,8 +168,36 @@ func setupControllers(mgr ctrl.Manager, enabledSchemes controllerv1.EnabledSchem
 		cfg := mgr.GetConfig()
 		volcanoClientSet := volcanoclient.NewForConfigOrDie(cfg)
 		gangSchedulingSetupFunc = common.GenVolcanoSetupFunc(volcanoClientSet)
+		gvk := schema.GroupVersionKind{
+			Group:   v1beta1.GroupName,
+			Version: v1beta1.GroupVersion,
+			Kind:    "PodGroup",
+		}
+		_, err := mgr.GetRESTMapper().RESTMapping(gvk.GroupKind(), gvk.Version)
+		if err != nil {
+			if meta.IsNoMatchError(err) {
+				setupLog.Error(err, "crd might be missing, please install volcano podGroup crd.")
+				os.Exit(1)
+			}
+			setupLog.Error(err, "unable to get volcano podGroup crd")
+			os.Exit(1)
+		}
 	} else if gangSchedulerName != "" {
 		gangSchedulingSetupFunc = common.GenSchedulerPluginsSetupFunc(mgr.GetClient(), gangSchedulerName)
+		gvk := schema.GroupVersionKind{
+			Group:   scheduling.GroupName,
+			Version: "v1alpha1",
+			Kind:    "PodGroup",
+		}
```

Comment:
Import `sigs.k8s.io/scheduler-plugins/apis/scheduling/v1alpha1`, and then 

```suggestion
		gvk := scheduling.SchemeGroupVersion.WithKind("PodGroup")
```